mtp_cashbook/templates/policy-change-warning-credits.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n %}
+
+<div class="mtp-notification mtp-notification--info">
+  <h2 class="mtp-notification__headline">
+    {% trans 'No more bank transfers' %}
+  </h2>
+  <div class="mtp-notification__message" style="display:block">
```

Comment:
I don't like this inline style, however it's been done before and would need a tmp-common change to remove it in a sensible way, so living with it for now, is that fair?

Reply:
yeah it's been done for "temporary" banners in send-money. so far, i don't think we've got any that are expected to be around after nov 2, right?

Reply:
actually, i can't remember if some variation of this banner is supposed to remain after nov 2. if so, you could add a new sibling class to `mtp-notification__message` to force the banner open if you wanted? (either in common or here)

Reply:
Actually, looking at the CSS I can just include the class that is toggled by JS and force the box to be open without the inline style, updated.

Reply:
oh cool does that work? i thought i tried it in send-money, but it looked funny; clearly didn't try hard enough!